packages/web-js-sdk/src/sdk/flow.ts

```diff
@@ -4,7 +4,13 @@ import { isSupported } from './webauthn';
 type CoreSdkFlowStartArgs = Parameters<CoreSdk['flow']['start']>;
 type Options = Pick<
   CoreSdkFlowStartArgs[1],
-  'tenant' | 'redirectUrl' | 'redirectAuth' | 'oidcIdpStateId'
+  | 'tenant'
+  | 'redirectUrl'
+  | 'redirectAuth'
+  | 'oidcIdpStateId'
+  | 'samlIdpStateId'
+  | 'samlIdpUsername'
```

Comment:
I don't see this prop in the server PR https://github.com/descope/orchestrationservice/pull/1416/files#diff-ca68e226af5f6291b8d818dca0420419518228c1f47965633c0ad81ce3d1ca4dR172

Reply:
its getting to the flow by redirecting from onetime service.. thats why..